ibm_catalog.json

```diff
@@ -0,0 +1,216 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-ease",
+        "label": "Cloud automation for Enterprise Application Service",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution"
```

Comment:
your going to have to add one valid tag for the type. You can run `ibmcloud catalog filter options --all` to see all tags. Suggest to use same tag as the service itself which is probably `enterprise_app` ?